File: src/IsbInterop/Tasks/Wrappers/CustomJobInfo.vb
Imports IsbInterop.Base

Namespace Tasks.Wrappers

  ''' <summary>
  ''' Обертка над ICustomJobInfo.
  ''' </summary>
  Friend Class CustomJobInfo
    Inherits CustomWorkInfo
    Implements ICustomJobInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwICustomJobInfo">COM-объект ICustomJobInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwICustomJobInfo As Object, scope As IScope)
      MyBase.New(rcwICustomJobInfo, scope)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop.CSharp/Tasks/Wrappers/CustomJobInfo.cs
using IsbInterop.Base;

namespace IsbInterop.Tasks.Wrappers
{
  /// <summary>
  /// Обертка над ICustomJobInfo.
  /// </summary>
  internal class CustomJobInfo : CustomWorkInfo, ICustomJobInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwICustomJobInfo">COM-объект ICustomJobInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public CustomJobInfo(object rcwICustomJobInfo, IScope scope)
      : base(rcwICustomJobInfo, scope) { }
  }
}
